draft-packages/tooltip/KaizenDraft/Tooltip/index.ts

```diff
@@ -1 +1 @@
-export { default as Tooltip, TooltipProps } from "./Tooltip"
+export { Tooltip, TooltipProps } from "./Tooltip"
```

Comment:
We could do this, but not a big deal currently.

```suggestion
export * from "./Tooltip"
```